docs/usage_general.rst.inc

```diff
@@ -236,6 +236,15 @@ If you are suspicious whether your Borg repository is still consistent
 and readable after one of the failures mentioned above occurred, run
 ``borg check --verify-data`` to make sure it is consistent.
 
+.. rubric:: Requirements for Borg repository file systems
+
+- Long file names
+- At least three directory levels with short names
+- File sizes up to a few hundred MB
```

Comment:
No, internal specificities shouldn't go in here. Rather, "- Typically, file sizes up to a few hundred MBs. Large repositories may require large files (>2 GB)." (i.e. not FAT32 for a big repo)